clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -71,11 +66,13 @@
   @Getter private final List<String> omitNamespaces;
   private final List<KubernetesKind> kinds;
   private final Map<KubernetesKind, InvalidKindReason> omitKinds;
+  private final Map<KubernetesKind, InvalidKindReason> omitKindsComputed = new HashMap<>();
```

Comment:
Since we're not modifying `omitKinds` anymore and just leaving it to represent the configured omit kinds, it can probably just be a `List<String>` and the constructor can just grab the value from `managedAccount.getOmitKinds()` without any post-processing.

It's a bit confusing now when we need to check one vs the other, so I'd suggest just populating `omitKindsComputed` with `omitKinds` in the constructor and just always using `omitKindsComputed` going forward.  The original `omitKinds` would exist only for determining if there was any change.